app/src/main/kotlin/com/saltedge/authenticator/features/main/MainActivityViewModel.kt

```diff
@@ -141,6 +141,15 @@ class MainActivityViewModel(
         }
     }
 
+    fun onActivityStart(intent: Intent?) {
+        if (intent?.getBooleanExtra(KEY_CLEAR_APP_DATA, false) == true) {
+            interactorV1.sendRevokeRequestForConnections()
+            interactorV2.sendRevokeRequestForConnections()
```

Comment:
redundant call . `interactorV1` have wiped application before. 
You can merge both interactors into one.